terraform/application/variables.tf

```diff
@@ -59,6 +59,26 @@ variable "enable_monitoring" {
   description = "Enable monitoring and alerting"
 }
 
+variable "webapp_memory_max" {
+  type    = string
+  default = "1Gi"
+}
+
+variable "webapp_replicas" {
+  type = number
+  default = 1
+}
+
+variable "worker_memory_max" {
+  type    = string
+  default = "1Gi"
+}
+
+variable "worker_replicas" {
+  type = number
+  default = 1
+}
+
```

Comment:
also postgres_enable_high_availability and postgres_flexible_server_sku need to be declared

Reply:
ah of course, thanks 🤦🏽